pkg/graphviz/traceflow_test.go

```diff
@@ -295,13 +511,145 @@ Delivered", shape=box, style="rounded,filled,solid" ];
+	cluster_source_1 [ color="#696969", fillcolor="#DCDCDC", label="SpoofGuard
+Forwarded", shape=box, style="rounded,filled,solid" ];
+	cluster_source_2 [ color="#696969", fillcolor="#DCDCDC", label="Forwarding
+Output
+Forwarded
+Tunnel Destination IP : 192.168.77.100", shape=box, style="rounded,filled,solid" ];
+
+}
+;
+	subgraph cluster_destination {
+	bgcolor="#F8F8FF";
+	label="k8s-node-control-plane";
+	labeljust=r;
+	style="filled,bold";
+	"10.10.0.2" [ color="#808080", fillcolor="#C8C8C8", style="filled,bold" ];
+	capturedPacket [ color="#696969", label="Captured Packet:\lSource IP: 10.10.1.4\lDestination IP: 10.10.0.2\lLength: 84\lIPv4 Header: \l    Flags: 2\l    Protocol: 1\l    TTL: 63\lTransport Header: \l    TCP: \l        Source Port: 123\l        Destination Port: 1\l", shape=note, style="rounded,filled,solid" ];
+	cluster_destination_1 [ color="#696969", fillcolor="#DCDCDC", label="Forwarding
+Output
+Delivered", shape=box, style="rounded,filled,solid" ];
+	cluster_destination_2 [ color="#696969", fillcolor="#DCDCDC", label="Forwarding
+Classification
+Received", shape=box, style="rounded,filled,solid" ];
+
+}
+;
+
+}
+`
+
+	expectedOutputWithNoliveTf := `digraph G {
+	center=true;
+	label="test-No-live-traceflow";
+	labelloc=t;
+	->cluster_source_1[ color="#C0C0C0", dir=forward, minlen=1, penwidth=2.0 ];
+	cluster_source_1->cluster_source_2[ color="#C0C0C0", dir=forward, minlen=1, penwidth=2.0 ];
+	->cluster_destination_1[ color="#C0C0C0", dir=back, minlen=1, penwidth=2.0, style="invis" ];
+	cluster_destination_1->cluster_destination_2[ color="#C0C0C0", dir=back, minlen=1, penwidth=2.0 ];
+	cluster_source_2->cluster_destination_2[ color="#C0C0C0", constraint=false, dir=forward, penwidth=2.0 ];
+	subgraph cluster_source {
+	bgcolor="#F8F8FF";
+	label="k8s-node-worker-1";
+	labeljust=l;
+	style="filled,bold";
+	[ color="#808080", fillcolor="#C8C8C8", style="invis" ];
+	cluster_source_1 [ color="#696969", fillcolor="#DCDCDC", label="SpoofGuard
+Forwarded", shape=box, style="rounded,filled,solid" ];
+	cluster_source_2 [ color="#696969", fillcolor="#DCDCDC", label="Forwarding
+Output
+Forwarded
+Tunnel Destination IP : 192.168.77.100", shape=box, style="rounded,filled,solid" ];
+
+}
+;
+	subgraph cluster_destination {
+	bgcolor="#F8F8FF";
+	label="k8s-node-control-plane";
+	labeljust=r;
+	style="filled,bold";
+	[ color="#808080", fillcolor="#C8C8C8", style="invis" ];
+	cluster_destination_1 [ color="#B22222", fillcolor="#EDD5D5", label="Forwarding
+Output
+Dropped", shape=box, style="rounded,filled,solid" ];
+	cluster_destination_2 [ color="#696969", fillcolor="#DCDCDC", label="Forwarding
+Classification
+Received", shape=box, style="rounded,filled,solid" ];
+
+}
+;
+
+}
+`
+
 	tests := []struct {
 		name           string
 		traceflow      crdv1alpha1.Traceflow
 		expectedOutput string
 	}{
 		{
-			name:           "live traceflow with source Pod",
+			name:           "live traceflow with source pod",
```

Comment:
Why did you change Pod to pod?
Usually we would use upper camel case for K8s resources.
Please check https://github.com/tnqn/code-review-comments#write-proper-nouns-formally